lxd/storage_volumes_snapshot.go

```diff
@@ -640,6 +673,25 @@ func autoCreateCustomVolumeSnapshotsTask(d *Daemon) (task.Func, task.Schedule) {
 				continue
 			}
 
+			// If there is more than one node (clustering), a stable random node is chosen to perform the snapshot.
+			if len(availableNodeIDs) > 1 {
```

Comment:
@monstermunchkin I think this decision should still be using the count of `nodes` (even offline ones) to indicate whether we are clustered or not, as if there are no online nodes (or there is some lookup problem, see my comment above) then the snapshot will occur on all nodes as it presumes its non-clustered.

Reply:
Fixed